packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/RequestLineEdit.tsx

```diff
@@ -1,12 +1,31 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import {
   ModalMode,
   useDialog,
   DialogButton,
   BasicSpinner,
   useTranslation,
+  useBufferState,
+  RecordPatch,
+  generateUUID,
 } from '@openmsupply-client/common';
 import { ItemRowFragment } from '@openmsupply-client/system';
+import { RequestLineEditForm } from './RequestLineEditForm';
+import {
+  useSaveRequestLines,
+  useRequestRequisitionFields,
+  useRequestRequisitionLines,
+  useIsRequestRequisitionDisabled,
+  RequestRequisitionLineFragment,
+} from '../../api';
+
+export type DraftRequestRequisitionLine = Omit<
```

Comment:
Just like the other modals, creating a draft line which can be saved later. This is simplified in that there is just one line per item